assets/js/admin.js

```diff
@@ -1119,6 +1119,36 @@
       return str === null || str.match(/^ *$/) !== null;
     }
 
+        // Event listener for the 'Check Terms' button click
+        $('.auto-terms-content.st-delete-unuused-terms').find('#check-terms-btn').on('click', function(e) {
+          e.preventDefault();
+        
+          var numberRarely = $('#number-delete').val();
+        
+          $('#terms-feedback').text(st_admin_localize.checking_terms_message); 
+        
+          $.ajax({
+            url: st_admin_localize.ajaxurl,
+            method: "POST",
+            data: {
+                action: 'check_terms',
```

Comment:
**check_terms** is too generic for an action name, check other ajax action on the code and update your ajax request action name to follow same patter as others to avoid conflict with other plugins